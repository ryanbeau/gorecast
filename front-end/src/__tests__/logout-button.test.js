import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import React from "react";
import LogoutButton from "../components/logout-button.js";


it("renders correctly", () => {
  const wrapper = shallow(<LogoutButton />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
