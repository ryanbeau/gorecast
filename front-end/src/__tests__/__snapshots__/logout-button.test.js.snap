// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="btn btn-danger btn-block"
  onClick={[Function]}
>
  Log Out
</button>
`;
